setup.cfg

```
@@ -43,9 +43,23 @@ python_requires = >= 3.7
 [options.extras_require]
 dev =
     setuptools>=3.7.1
+    mypy>=0.931
+    pandas>=1.3.5
+
+pandas =
+    pandas>=1.3.5
 keepalive =
     keepalive>=0.5
 
 [options.entry_points]
 console_scripts =
     rqw = SPARQLWrapper.main:main
+
+[mypy]
```

Best to be explicit about what directories are type checked:

```suggestion
[mypy]
files = SPARQLWrapper
```